components/conjunctions/_screen-md-min.scss

```diff
@@ -35,23 +35,23 @@
 			}
 		}
 
-		[class*="cnjnctn-col"]:not(:last-child) {
+		> [class*="cnjnctn-col"]:not(:last-child) {
```

Comment:
You have `> [class*="cnjnctn-col"]` twice. They could be combined and then have `&:not(:first-child)` and `&:not(:last-child)` as subrules.